src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -18,6 +18,7 @@ CA1846 | Performance | Info | PreferAsSpanOverSubstring, [Documentation](https:/
 CA1847 | Performance | Info | UseStringContainsCharOverloadWithSingleCharactersAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1847)
 CA1848 | Performance | Hidden | LoggerMessageDefineAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1848)
 CA2017 | Reliability | Warning | LoggerMessageDefineAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2017)
+CA2018 | Reliability | Info | UseAsyncMethodInAsyncContext, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2018)
```

Comment:
Is `Reliability` the correct category? Or would `Performance` be more appropriate?

Reply:
https://github.com/dotnet/runtime/issues/45661#issuecomment-765702030
@carlossanlop had suggested `Reliability` as the category in the api review

Reply:
Thanks. @carlossanlop / @stephentoub -- What do you two think about _Reliability_ vs. _Performance_ on this analyzer? The category was not discussed during the API review.

Reply:
It's a bit of a gray area, but I'd go with performance.  Typically the negative impact of using a sync method from an async method is reduced scalability, which is closer to performance than reliability.  In a bad version of this, that sync method is actually doing async work and blocking waiting for it to complete, which brings it back closer to reliability (as that can lead to starvation and the like), but the problem there is the sync-over-async nature of the sync operation, not the fact that the async method is using the sync method.